src/routes/libraries.js

```diff
@@ -17,10 +19,22 @@ const maxQueryLength = 512;
  * @return {Promise<Object[]>}
  */
 const browse = async (query, searchFields) => {
+    // Normalize the search fields
+    const fields = searchFields.filter(field => validSearchFields.includes(field)).sort();
+
+    // Check if we have a cached result for this query
+    const cacheKey = await crypto.subtle.digest(
+        { name: 'SHA-512' },
+        new TextEncoder().encode(`${query}:${fields.join(',')}`),
+    ).then(buf => `libraries:${Array.from(new Uint8Array(buf)).map(b => b.toString(16).padStart(2, '0')).join('')}`);
+    const cached = await CACHE.get(cacheKey, { type: 'json' });
+    if (cached) return cached;
```

Comment:
Is it idiomatic javascript to not use `{ }`?

The background behind the ask is that it adds no material value other than looking slightly simpler, but conversely it can very easily lead to bugs further down the line when using multiline statements and forgetting to add the brackets. It's also a type of bug that code formatters/linters cannot easily detect.

In other words, the cost (risk of bugs) is high but the value (looks a bit nicer for some contributors) is low.
I, like many others, have been bitten by bugs due to this code style. It's generally not a matter of if, but a matter of when. I generally prefer to write code in a way that reduces the potential for bugs, and using `{ }` always is part of that. 

In fact, modern languages such as Rust or Swift do not have the ability to remove `{ }` for one-liners due to the high risk/low value, and to the many bugs and security issues that it has created over time (such as this SSL bug: https://www.imperialviolet.org/2014/02/22/applebug.html)

Reply:
Conventionally I've always gone for not using curlys when using a single-statement if. This is very similar to the standard styling applied by [Prettier](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEBLAZgAgBTwM4wCUmkUeEANnAHQUQDm2hA3ADpTsY75HuYnRyVWgyZsOULrjgFiwAWUo06jFpgC+7TlmmzMwPgqHLRLdupAAaEBAAOMVIOSgAhgCc3EAO4AFdwjxkEBcKLxcAT0DrACM3FzAAazgYAGUXAFs4ABlUKDhkdBC8OBi4xOSU23jc+mQYNwBXEpA4dOi4ABMOzqyXKHoGl3o4ADEIN3SXGAd+oJcGmAgrEAALGHSKAHUV1HwqsDgUgN3UADdd8KCwPCi0Mjg3GB84+kmCouaAKzwADxSaqgARQaEHg7woxWsVTcxTcQWiLnaFGWtjcuRgm1QHRgK2QAA4AAxQzzFTZxWxBVEyB6nfLWACOIPgzzsgRQLjwAFo8p1Oss3HBGagBc8hm8kIUIc1iulUHVGtKAXBgaD8hKPtYYIjMdjcUgAEyauKoCg1ADCEHS4paeAArMsGsUACqItmSyEgU5NACSUG6sBSYDR9gAgn6UjBwlRwcV1OogA)